khmer/_khmermodule.cc

```diff
@@ -466,6 +466,20 @@ ReadParser_iter_reads(PyObject * self, PyObject * args )
     return PyObject_SelfIter( self );
 }
 
+static
+PyObject *
+ReadParser_get_num_reads(khmer_ReadParser_Object * me, PyObject * args)
+{
```

Comment:
Hang on, how about using `getsetters`, *a la* the `HLLCounter`'s `alpha` and `error_rate`? From what I can see, it would be difficult to expose a protected member of the IParser class via the khmer_ReadParser_Object struct.

However, would love to stand corrected